requirements/dev.txt

```diff
@@ -1,4 +1,5 @@
 black==19.10b0
 coverage==4.5.4
+pre-commit==2.0.1
 pytest==5.3.5
```

Comment:
We could separate this libraries on a different file: `test.txt` as it tends to grow up: `pytest-mock`, `pytest-dist`, `pytest-cov` and so on...

Reply:
I will do it in a new PR.